src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `book_shop` /*!40100 DEFAULT CHARACTER SET utf8mb3 */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+CREATE TABLE `books` (
+                         `id` int NOT NULL AUTO_INCREMENT,
```

Comment:
When creating a table in MySQL, use bigint column type for storing id.
Primary key by default not null

```suggestion
                         `id` bigint AUTO_INCREMENT,
```